.github/workflows/docker-release.yml

```diff
@@ -0,0 +1,52 @@
+name: Daily build
+on:
+  schedule:
+    - cron: "0 1 * * *" # Every day at 1:00 AM
+  workflow_dispatch: {}
+jobs:
+  release_image:
+    name: Build and publish docker image
+    runs-on: ubuntu-latest
+    env:
+      IMAGE_REPOSITORY: mongodb/atlas
+      PLATFORMS: linux/amd64,linux/arm64
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Set date
+        id: set-date
+        run: |
+          DATE=$(date +'%Y-%m-%d')
+          echo DATE=${DATE} >> $GITHUB_ENV
+      - name: 'Get latest tag'
+        id: get-latest-tag
+        run: echo "::set-output name=tag::$(git tag --list --sort=-version:refname "atlascli*" | head -n 1)"
```

Comment:
If RC versions are going to happen they also going to be picked up. We do have only 2 rc versions so risks are low? Are we ok to accept that? Maybe we can use github releases action to fetch latest release?

https://github.com/bruceadams/get-release

This will also ensure that tags without releases (drafts) aren't covered.

Reply:
Updated using this action now: https://github.com/marketplace/actions/get-latest-release
This was my initial plan but was developing on my fork where I did not have the releases. I agree this should be more reliable.